share/admin/__init__.py

```diff
@@ -21,18 +21,23 @@
 
 from oauth2_provider.models import AccessToken
 
-from share.robot import RobotAppConfig
+from share.admin.readonly import ReadOnlyAdmin
+from share.admin.share_objects import CreativeWorkAdmin
+from share.models.banner import SiteBanner
 from share.models.celery import CeleryTask
 from share.models.change import ChangeSet
 from share.models.core import NormalizedData, ShareUser
+from share.models.creative import AbstractCreativeWork
 from share.models.ingest import RawDatum, Source, SourceConfig, Harvester, Transformer
 from share.models.logs import HarvestLog
 from share.models.registration import ProviderRegistration
-from share.models.banner import SiteBanner
-from share.readonlyadmin import ReadOnlyAdmin
```

Comment:
Completely misread that, sorry. Brain turned that into "Looks like only readonlyadmin".
Will remove.